cc/tiles/picture_layer_tiling_set.cc

```diff
@@ -274,7 +274,7 @@ void PictureLayerTilingSet::CleanUpTilings(
       continue;
 
     // Don't remove tilings that are required.
-    if (base::Contains(needed_tilings, tiling.get())) {
+    if (base::Contains(needed_tilings, tiling.get()->tile_id)) {
```

Comment:
Not sure how this works?  You're changing the usage of `base::Contains` to pass an int instead of a pointer, but I don't see a matching implementation that would accept that?  But if it builds...

Reply:
base::Contains is a templatized function that will work with most (?) collections of values, so I didn't need to change anything to get this to work.